code/index.html

```diff
@@ -1,22 +1,137 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
+    <title>Tre Punkter</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@300;400;500&display=swap" rel="stylesheet">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
+
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <nav>
+        <!---------------NAVBAR--------------->
+        <input type="checkbox" id="check">
+        <label for="check" class="checkbtn"><i class="fas fa-bars"></i></label>
+        <img class="logo" src="IMG/TP-Logo-light.png" alt="TrePunkter logo">
+        <ul>
+          <li><a class="active" href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Services</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+    <section class="big_news_container">
+      <div class="big_news_part_1">
+        <img class="news_image" src="IMG/steveand2.jpg" alt="">
```

Comment:
You could write something here under alt in case the image doesn't load. (I see you've done that on other places) :)